css/device-catalog.css

```diff
@@ -0,0 +1,884 @@
+    font-size: 18px;
+    line-height: 24px;
+}
+.side-bar label {
+    position: relative;
+    padding-left: 43px;
+
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 14px;
+    line-height: 40px;
+}
+.filter-price {
+    max-width: 201px;
+    border-style: none;
+    padding: 0;
+    margin-bottom: 40px;
+    margin-top: 80px;
+}
+.filter-color {
+    max-width: 201px;
+    border-style: none;
+    padding: 0;
+    margin-bottom: 40px;
+}
+.filter-bluetooth {
+    max-width: 201px; 
+    border-style: none;
+    padding: 0;
+    margin-bottom: 40px;
+}
+.filter-button {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+.container-color input[type="checkbox"] {
+    opacity: 0;
+    position: absolute;
+}
+.container-color label::before {
+    content: "";
+    display: inline-block;
+    position: absolute;
+
+    width: 23px;
+    height: 23px;
+
+    border: 2px solid;
+
+    left: 0;
+    top: -2px; 
+}
+.container-color label::after {
+    content: "";
+    display: inline-block;
+    position: absolute;
+
+    width: 21px;
+    height: 12px;
+    border-left: 4px solid;
+    border-bottom: 4px solid;
+
+    transform: rotate(-45deg);
+    left: 4px;
+    top: -2px;
+}
+.container-color input[type="checkbox"] + label::after {
+    content: none;
+}
+.container-color input[type="checkbox"]:checked + label::after {
+    content: "";
+}
+.container-color input[type="checkbox"]:hover + label::before,
+.container-color input[type="checkbox"]:hover + label::after {
+    opacity: 0.6;
+}
+.container-color input[type="checkbox"]:focus + label::before,
+.container-color input[type="checkbox"]:focus + label::after {
+    opacity: 1;
+}
+
+.container-bluetooth input[type="radio"] {
+    position: absolute;
+    opacity: 0;
+}
+.container-bluetooth input[type="radio"] + label::before {
+    content: url(.//img/radio-off.svg);
+    display: inline-block;
+    position: absolute;
+    left: 0;
+    top: -2px; 
+}
+.container-bluetooth input[type="radio"]:checked + label::before {
+    content: url(.//img/radio-on.svg);
+    display: inline-block;
+    position: absolute;
+    left: 0;
+    top: -2px; 
+}
+.container-bluetooth input[type="radio"]:hover + label::before {
+    opacity: 0.6;
+}
+.container-bluetooth input[type="radio"]:focus + label::before {
+    opacity: 1;
+}
+.filter-button {
+    width: 200px;
+    background: linear-gradient(to top, transparent 16px, #f0c52e 16px, #f0c52e 24px, transparent 24px);
+    outline: none;
+    border: none;
+    padding-top: 8px;
+    padding-bottom: 8px;
+}
+.filter-button:hover {
+    background: #f0c52e;
+}
+.filter-button:active {
+    color: rgba(0, 0, 0, 0.3);
+}
+/* FOOTER */
+
+footer {
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 1px;
+    max-width: 1440px;
+    height: auto;
+    background-color: #464646; 
+}
+
+footer div {
```

Comment:
dangerous rule, bc all divs inside of footer will inherit it

Reply:
Done